template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/utils/BlueprintUtils.java

```diff
@@ -137,4 +140,11 @@ public Map<String, Object> prepareTags(JsonNode tags) throws com.fasterxml.jacks
     public boolean isSharedServiceReadyBlueprint(Blueprint blueprint) {
         return blueprint.getTags() != null && blueprint.getTags().getMap().containsKey("shared_services_ready");
     }
+
+    public BlueprintUpgradeOption getBlueprintUpgradeOption(JsonNode root) {
```

Comment:
Could you please cover with UT? Be sure to use JUnit 5, thanks.

Reply:
This is done in line 124 and 134 in the test class, Lajos.

Reply:
Sorry, which test class? I cannot see `BlueprintUtilsTest` being modified here. Actually this method does not seem to be used. Only `getBlueprintUpgradeOptionForGA()` is called (from `BlueprintV4RequestToBlueprintConverter`).